docs/platforms/javascript/common/best-practices/micro-frontends.mdx

```diff
@@ -21,22 +21,22 @@ keywords:
     "module metadata",
   ]
 ---
+<Alert>
+In order for the Sentry JavaScript SDK to work with your micro frontends, you'll need to either set up [module federation](https://module-federation.github.io/) or make sure that all your micro frontends are using the same version of the Sentry SDK.
+</Alert>
 
-If your frontend includes JavaScript bundles from multiple sources with
-different release cycles, you may want to identify these or route events to specific projects. This is especially useful if you've set up [module federation](https://module-federation.github.io/) or a similar frontend architecture.
+If your app uses micro frontends, it’s very useful to be able to track which one an error is coming from. To do this with Sentry, you can create either an automatic or a manual setup where you send events to separate Sentry projects representing each of your micro frontend services. This makes it easier to see what’s going wrong and where, helping you track issues and fix them faster, especially in complex frontend architectures.
```

Comment:
(feel free to disregard): I'd not use the term "service" as this is a rather backend- or data-related term as opposed to a UI/frontend module/application. I think it's fine to just refer to them as "micro frontend". 

```suggestion
If your app uses micro frontends, it’s very useful to be able to track which one an error is coming from. To do this with Sentry, you can create either an automatic or a manual setup where you send events to separate Sentry projects representing each of your micro frontends. This makes it easier to see what’s going wrong and where, helping you track issues and fix them faster, especially in complex frontend architectures.
```